README.md

```diff
@@ -1,5 +1,7 @@
 # depviz - dependency visualizer for the web
 
+> dependency visualizer for the web https://jbenet.github.io/depviz
```

Comment:
With abf5ee2b5 you capitalized “Dependency”.  But we still have the phrase repeated in both the H1 and the blockquote.  Can we drop “dependency visualizer for the web” from one or the other?